core/src/main/kotlin/maru/executionlayer/manager/ExecutionLayerManager.kt

```diff
@@ -88,11 +88,15 @@ interface ExecutionLayerManager {
 
   fun finishBlockBuilding(): SafeFuture<ExecutionPayload>
 
+  fun hasStartedBlockBuilding(): Boolean
```

Comment:
I believe you can't guarantee its consistency since the underlying node could restart any time. And I think the guarantees we can provide are as good as just assume that block building has started

Reply:
True, will try that use delayed block creator on first use approach. If that works I can remove this.